chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java

```diff
@@ -74,15 +75,15 @@ public void start(Stage stage) {
 
     @ParameterizedTest
     @EnumSource(LineStyle.class)
-    public void testRendererNominal(final LineStyle lineStyle) throws IOException, Exception {
+    public void testRendererNominal(final LineStyle lineStyle) throws Exception {
         for (ErrorStyle eStyle : ErrorStyle.values()) {
             renderer.setErrorType(eStyle);
             testRenderer(lineStyle);
         }
     }
 
     @Test
-    public void testRendererSepcialCases() throws IOException, Exception {
+    public void testRendererSepcialCases() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5764587)